test/unit_Hyperrectangle.jl

```diff
@@ -1,3 +1,6 @@
+import IntervalArithmetic
```

Comment:
```suggestion
using IntervalArithmetic
```

Reply:
Well this is the commit i made previously, but Travis complained..

Reply:
https://travis-ci.org/JuliaReach/LazySets.jl/jobs/503687286

Reply:
Then I suggest to remove this line completely and only get what you need.

Reply:
another solution is probably to do `using IntervalArithmetic` *and* also `using LazySets: radius` to avoid the conflict.

Reply:
You already have `using IntervalArithmetic: IntervalBox` one line below. So why do you need `IntervalArithmetic` at all?

Reply:
>  Then I suggest to remove this line completely and only get what you need.

How?

In the REPL one does `using LazySets, IntervalArithmetic` (without warnings) and can construct the `IntervalBox` using the common notation `IntervalBox(0..1, 0..1)` which requires `IntervalArithmetic`.

Otherwise, use the more verbose `IntervalBox(IntervalArithmetic.Interval(0, 1), IntevalArithmetic.Inteval(0, 1)`, which requires that the name `IntervalArithmetic` is available (this is the purpose of the `import IntervalArithmetic`).

Reply:
> So why do you need IntervalArithmetic at all?

Because `IntervalArithmetic.Interval` is undefined.

Reply:
Then let's do `using IntervalArithmetic: Interval`?

Reply:
Hmm this conflicts with LazySets's `Interval` type, and will cause problems if another test file uses lazysets' interval after this one. What is the argument against using `import`?

Reply:
... or otherwise, keep `using IntervalArithmetic` and also `using LazySets: radius` to avoid the namespace conflict. Both options look good to me.

Reply:
I don't understand this. The only difference between `import X` and `using X` [should be](https://docs.julialang.org/en/v1/manual/faq/#What-is-the-difference-between-%22using%22-and-%22import%22?-1) that in the first case one can write `y` instead of `X.y`. Why is there now a difference?

Reply:
Alright, the correct source is [here](https://docs.julialang.org/en/v1/manual/modules/#Summary-of-module-usage-1). `import` is correct here.